docs/index.asciidoc

```diff
@@ -193,7 +193,11 @@ can be either IP, HOST, IP:port, or HOST:port. The port defaults to
   * Value type is <<string,string>>
   * Default value is `"logstash-*"`
 
-The index or alias to search.
+The index or alias to search. See
```

Comment:
You can use a named link like so:

```
The index or alias to search. See https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-index.html[Elasticsearch Multi Indices documentation] for more information on how to reference
multiple indices.
```

This way we avoid having a full blown url in the documentation text

Reply:
I agree, thanks for noticing!